deploy/charts/harvester/templates/configmap.yaml

```diff
@@ -20,6 +20,21 @@ metadata:
 data:
   promote.sh: |-
     {{`KUBECTL="/host/$(readlink /host/var/lib/rancher/rke2/bin)/kubectl"
+    ROLE_LABELS="rke.cattle.io/control-plane-role=true rke.cattle.io/etcd-role=true"
+    ETCD_ONLY=false
+    if [[ -n "$1" ]]; then
```

Comment:
The code works now, but has chance to break:

if $1 == "rke.cattle.io/control-plane-role=true rke.cattle.io/etcd-role=true", the code will also assume `ETCD_ONLY`;  sure, the controller side does set such value at the moment.

Set `ETCD_ONLY` in following `case` statement will be better, uses exact match.